### PR TITLE
chore 144284347 (add): add files (files added for front end tests)

### PR DESCRIPTION
**Pull request by andela-oowonikoko**
#### What does this PR do?
- This PR sets up front-end tests

#### Description of Task to be completed?
- Setup front end tests

#### How should this be manually tested?
- This can be manually tested by checking my repo

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- Setup front end tests

#### Screenshots (if appropriate)
NA

#### Questions:PR Naming
N/A

### CODE REVIEW

File: client/src/components/document/DocumentForm.component.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Row, Input, Button } from 'react-materialize';
 const DocumentForm = ({ document, onSave, onChange, saving, errors }) => {
   return (    
     <form>
-      <h5>Create/Update a Document </h5>
+      <h5>Create/Update a Document</h5>
```
Comment (mradeybee) [inline client/src/components/document/DocumentForm.component.js:7]: Remove file if its not in use.

File: client/src/test/component/DocumentForm.spec.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  return shallow(<DocumentForm {...props} />);
+}
+
+describe('DocumentForm Test', () => {
+  it('renders form and h5', () => {
+    const wrapper = setup(false);
+    expect(wrapper.find('form').length).toBe(1);
+    expect(wrapper.find('h5').text()).toEqual('Create/Update a Document');
```
Comment (mradeybee) [inline client/src/test/component/DocumentForm.spec.js:22]: I am not sure if you are still using this, CHECK!

File: client/src/test/component/LoginForm.spec.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    onSubmit: () => {},
+    onChange: () => {}
+  };
+
+  return mount(<LoginForm {...props} />);
+}
+
+describe('LoginForm Test', () => {
+  it('renders form and h1', () => {
+    const wrapper = setup();
```
Comment (mradeybee) [inline client/src/test/component/LoginForm.spec.js:18]: DRY this up! 

move this to the describe block
```JavaScript
describe('LoginForm Test', () => {
    const wrapper = setup();
    // the it blocks
}
```

File: client/src/test/component/LoginForm.spec.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    expect(wrapper.find('form')).toExist;
+    expect(wrapper.find('input')).toExist;
+    expect(wrapper.find('h1').text()).toEqual('Login');
+    expect(wrapper.find('div')).toExist;
+    expect(wrapper.find('button')).toExist;
+  });
+
+  it('should take props', () => {
+    const wrapper = setup();
+    expect(wrapper.props().userLoginRequest).toExist;
```
Comment (mradeybee) [inline client/src/test/component/LoginForm.spec.js:28]: This is a function call, its better if you assign it to a variable E.G
```JavaScript
let wrapperProps = wrapper.props()

expect(wrapperProps.userLoginRequest).toExist;
expect(wrapperProps.onChange).toExist;
expect(wrapperProps.onSubmit).toExist;
```

**Comment by ftchirou** [comment]
Tests look good. A nitpick suggestion would be to use `.gitignore` to hide the `reports` folder and unnecessary files like `.DS_Store`.

File: client/src/test/component/ProfileForm.spec.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    getUser: () => {},
+    onSubmit: () => {},
+    onChange: () => {}
+  };
+
+  return mount(<ProfileForm {...props} />);
+}
+
+describe('ProfileForm Test', () => {
+  // it('should render self', () => {
```
Comment (mradeybee) [inline client/src/test/component/ProfileForm.spec.js:18]: Remove dead code!

File: selenium-debug.log [hunk 6]
```diff
@@ -0,0 +1,150 @@
+11:55:04.181 INFO - Selenium build info: version: '3.3.1', revision: '5234b32'
```
Comment (mradeybee) [inline selenium-debug.log:1]: Remove Log!

File: client/src/test/component/ProfileForm.spec.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import expect from 'expect';
+import React from 'react';
+import { mount, shallow } from 'enzyme';
```
[the remark this answers is not in the document]
Comment (mradeybee) [reply client/src/test/component/ProfileForm.spec.js:3]: @andela-oowonikoko This is a valid comment. Don't import `shallow` or `mount` if you are not using it.

**Comment by mradeybee** [comment]
@andela-oowonikoko You should remove every import of shallow and mount that you are not using. If you won't use it, don't import it.

**Pull request merged**